README.md

```diff
@@ -98,6 +98,13 @@ See that you can use the `--filter` --flag to filter the results by the package
 audit-tool index [bundles|packages|channels] --index-image=registry.redhat.io/redhat/redhat-operator-index:v4.5 --filter="mypackagename"
 ```
 
+### Option to run in dedicated environments
+
+Use the flag `--server-mode` to generate the reports in dedicated environments. By using this flag option the images
+which are downloaded will not be removed which will allow the reports to be generate faster after the first execution. 
+
+Also, ensure that you have enough space to store all images. Note that the default behavior is to remove them, when this option is not used.  
```

Comment:
Might want to give a size guess so admins know how much space to allocate.

Reply:
Then that is up to the images that will be used.